internal/controller/api/v1beta1/agent_types.go

```diff
@@ -168,54 +130,21 @@ type HostProgressInfo struct {
 	StageUpdateTime string `json:"stageUpdateTime,omitempty"`
 }
 
-type L2Connectivity struct {
-	OutgoingIPAddress string `json:"outgoingIPAddress,omitempty"`
-	OutgoingNic       string `json:"outgoingNIC,omitempty"`
-	RemoteIPAddress   string `json:"remoteIPAddress,omitempty"`
-	RemoteMac         string `json:"remoteMAC,omitempty"`
-	Successful        bool   `json:"successful,omitempty"`
-}
-
-type L3Connectivity struct {
-	OutgoingNic     string `json:"outgoingNIC,omitempty"`
-	RemoteIPAddress string `json:"remoteIPAddress,omitempty"`
-	Successful      bool   `json:"successful,omitempty"`
-}
-
-type HostConnectivityValidationInfo struct {
-	HostDeploymentName *AgentReference `json:"hostDeploymentName"`
-	L2Connectivity     L2Connectivity  `json:"l2Connectivity,omitempty"`
-	L3Connectivity     L3Connectivity  `json:"l3Connectivity,omitempty"`
-}
-
 type HostNTPSources struct {
 	SourceName  string             `json:"sourceName,omitempty"`
 	SourceState models.SourceState `json:"sourceState,omitempty"`
 }
 
 // AgentStatus defines the observed state of Agent
 type AgentStatus struct {
-	State     string `json:"state,omitempty"`
-	StateInfo string `json:"stateInfo,omitempty"`
-	// Name in REST API: status_updated_at
-	StateUpdatedTime *metav1.Time `json:"stateUpdatedTime,omitempty"`
-	// Name in REST API: logs_collected_at
-	LogsCollectedTime *metav1.Time `json:"logsCollectedTime,omitempty"`
-	InstallerVersion  string       `json:"installerVersion,omitempty"`
-	// Name in REST API: updated_at
-	UpdateTime *metav1.Time `json:"updateTime,omitempty"`
-	// Name in REST API: checked_in_at
-	CheckedInTime         *metav1.Time                     `json:"checkedInTime,omitempty"`
-	Hostname              string                           `json:"hostname,omitempty"`
-	Bootstrap             bool                             `json:"bootstrap,omitempty"`
-	DiscoveryAgentVersion string                           `json:"discoveryAgentVersion,omitempty"`
-	Inventory             HostInventory                    `json:"inventory,omitempty"`
-	ValidationInfo        HostValidationInfo               `json:"hostValidationInfo,omitempty"`
-	Progress              HostProgressInfo                 `json:"progress,omitempty"`
-	Connectivity          []HostConnectivityValidationInfo `json:"connectivity,omitempty"`
-	APIVipConnectivity    bool                             `json:"apiVIPConnectivity,omitempty"`
-	NtpSources            []HostNTPSources                 `json:"ntpSources,omitempty"`
-	Conditions            []conditionsv1.Condition         `json:"conditions,omitempty"`
+	State      string                   `json:"state,omitempty"`
```

Comment:
doesn't it move to debug info?

Reply:
Yes, I will rebase on top of #1972 once merged

Reply:
Done